plugins/modules/netbox_service.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Kulakov Ilya  (@TawR1024)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: netbox_service
+short_description: Creates or removes service from Netbox
+description:
+  - Creates or removes service from Netbox
+notes:
+  - This should be ran with connection C(local) and hosts C(localhost)
+author:
+  - Mikhail Yohman (@FragmentedPacket)
+  - Kulakov Ilya (@TawR1024)
+requirements:
+  - pynetbox
+version_added: '2.8'
+options:
+  netbox_url:
+    description:
+      - URL of the Netbox instance resolvable by Ansible control host
+    required: true
+    type: str
+  netbox_token:
+    description:
+      - The token created within Netbox to authorize API access
+    required: true
+    type: str
+  data:
+    description:
+      - Defines the service configuration
+    suboptions:
+      device:
+        description:
+          - Specifies on which device the service is running
+        type: raw
+      virtual_machine:
+        description:
+        type: str
+      port:
+        description:
+          - Specifies which port used by service
+        type: int
+      protocol:
+        description:
+          - Specifies which protocol used by service
+        type: int
+        choises:
```

Comment:
Fix the typo: choises -> choices

And then just have the choices as the words and not integers since we do that in the background, even those they are valid at the moment.